File: WeAre.Athenaeum.React.Components/src/models/server/ApplicationSettings.ts
import {IApplicationSettings} from "@weare/athenaeum-react-common";

export default class ApplicationSettings implements IApplicationSettings {
    public googleMapApiUrl: string = "";

    public googleMapApiKey: string = "";

    public defaultConstructionSiteExternalId: number = 0;

    public hoursPrice: number = 0;

    public mileagePrice: number = 0;

    public isApplicationSettings: true = true;
}

File: Renta.TestApplication.WebUI.Server/Models/ApplicationSettings.cs
using System;
using WeAre.Athenaeum.TemplateApp.Common.Configuration;

namespace WeAre.Athenaeum.TemplateApp.WebUI.Server.Models
{
    public sealed class ApplicationSettings
    {
        #region Constructors
        
        public ApplicationSettings()
        {
        }

        public ApplicationSettings(TestApplicationConfiguration configuration)
        {
            if (configuration == null)
                throw new ArgumentNullException(nameof(configuration));

            GoogleMapApiUrl = configuration.GoogleSettings.MapApiUrl;
            GoogleMapApiKey = configuration.GoogleSettings.MapApiFeKey;
        }
        
        #endregion
        
        #region Properties
        
        public string GoogleMapApiKey { get; set; }

        public string GoogleMapApiUrl { get; set; }

        public bool IsApplicationSettings => true;

        #endregion
    }
}